src/mon/Monitor.cc

```diff
@@ -3228,6 +3228,8 @@ void Monitor::handle_command(MonOpRequestRef op)
     mgrstatmon()->dump_info(f.get());
 
     paxos->dump_info(f.get());
+    
+    f->dump_object("servicemap", mgrstatmon()->get_service_map());
```

Comment:
@david-z if you believe that servicemap should be included by the output of "ceph report", probably you need to update `MgrStatMonitor::dump_info()` instead.

Reply:
Good idea. I didn't notice this because my change was based on luminous which didn't have `MgrStatMonitor::dump_info()`. 

This PR might not need to backport, so I will update it as your suggestion.